programs/hyperplane/src/utils/macros.rs

```diff
@@ -88,3 +88,14 @@ macro_rules! emitted {
         return Ok($event);
     };
 }
+
+/// Macro to convert a value to u64, with useful error message
+#[macro_export]
+macro_rules! to_u64 {
+    ($val: expr) => {
+        u64::try_from($val).map_err(|_| {
+            msg!("Unable to convert {} to u64: {}", stringify!($val), $val);
```

Comment:
I would add file and line:
```suggestion
            msg!("[{}:{}] Unable to convert {} to u64: {}", file!(), line!(), stringify!($val), $val);
```

wdyt?

Reply:
I will leave it just because it already prints this with the `error!` macro